### DISCUSSION TITLE
Marp website builder

### DISCUSSION THREAD

**Discussion by tristan-f-r** [marp-team/marp#393] [Ideas] [answered]
Given the fact that marp is built on languages that can run on the web, would it be achievable to create a simple online editor that allows you to upload md files and use them to make a presentation?

**Comment by yhatt** [comment, marked as answer]
Of course. It's reason why our ecosystem is open, including [the framework](https://github.com/marp-team/marpit) and [the core](https://github.com/marp-team/marp-core).

- [Marp Web](https://github.com/marp-team/marp-web) demo (no longer maintained): https://web.marp.app/
- [TheodoreChu/marp-editor](https://github.com/TheodoreChu/marp-editor): https://demo.marpeditor.com/
